<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transaction</class>
 <widget class="QDialog" name="Transaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaction</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 233, 235);
font: 9pt &quot;Courier&quot;;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="savingsLabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>50</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Savings:</string>
   </property>
  </widget>
  <widget class="QLabel" name="loanLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>onLoan</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Transaction Type</string>
   </property>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Courier&quot;;
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="witthdrawButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Courier&quot;;</string>
    </property>
    <property name="text">
     <string>Withdraw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loanButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>142</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Loan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="payBackButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Pay back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otherButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="savingLcd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="digitCount">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="loanLcd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="digitCount">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="nameLable">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Transaction</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
